cliquet_docs/contributing.rst

```diff
@@ -84,6 +84,41 @@ to speed-up the execution:
     nosetests -s --with-mocha-reporter cliquet.tests.test_views_hello:HelloViewTest.test_returns_info_about_url_and_version
 
 
+Load tests
+----------
+
+A load test is provided in order to run end-to-end tests on *Cliquet* through a sample application,
+or prevent regressions in terms of performance.
+
+The following ``make`` command will check briefly the overall sanity of the API,
+by running a server and running a very few random HTTP requests on it.
+
+::
+
+    make loadtest-check-simulation
+
+It is possible to customize this load test, by focusing on a single action,
```

Comment:
nit: I believe that we are trying to use the english form elsewhere in the documentation (for the "z" versus "s"). This should be "customise", then.